bin/p-s.js

```diff
@@ -55,7 +56,29 @@ runPackageScript({
 function getPSConfig() {
   var psConfigFilename = program.config ? resolve(process.cwd(), program.config) : findUp.sync('package-scripts.js')
   try {
-    return require(psConfigFilename)
+    // Require a module like babel-register
+    if (program.require) {
+      var requirePath = (program.require.substr(0, 1) === '.'
+        ? resolve(process.cwd(), program.require)
+        : program.require)
+
+      try {
+        require(requirePath)
+      } catch (e) {
+        log.warn({
+          message: colors.yellow('Unable to preload ' + program.require),
+          ref: 'unable-to-preload-module'
+        })
+      }
+    }
+
+    var config = require(psConfigFilename)
+    // babel-register is loading the default package into a object with the prop
+    // default so if it exist use it as the config
+    if (config.default) {
```

Comment:
Maybe we should check `if (config.__esModule === true)` instead of `if (config.default)`?